cockroachdb/docs
# Force pretty URLs on all pages within the docs

Minimum testing plan:

1. Navigate to a page from the sidenav.
2. Follow the links to other pages.
3. Use the version switcher to go to another version of that page.
4. Open the v23.1 releases. Find the link for feature highlights (e.g., `releases/v23.1#v23-1-0-feature-highlights`). Find the link called `what's new in v23.1 in our docs`. Click it, and make sure search works.
5. Click on a result.
6. Search the docs for `install cockroachdb`. Click on one of the links.
7. Click on `Install CockroachDB` in the sidenav to ensure the redirect works correctly.
8. Navigate as many links in the docs ad nauseam.

Additional items to implement:

1. URL rewrites from `/index.html` to `/` and from `.html` to `/` for both `<a>` tags and canonicals
